docs/release-notes/allow-and-require-user-passwords.rst

```diff
@@ -0,0 +1,29 @@
+:orphan:
+
+**Breaking Changes**
+
+-  Python SDK and CLI: Enforce password requirements for all non-remote users, aligning with WebUI
+   password standards and having the following requirements:
+
+      -  Passwords must be at least 8 characters long (and not None).
```

Comment:
I don't see any documentation in the actual docs about the password requirements. I think it's important enough that it should go out along with this change.

Once that's done, then the Breaking Changes section would ideally link to the page in docs (@tara-det-ai: what do you think about https://docs.determined.ai/latest/manage/users.html?) with that information.

Reply:
Looks good!